src/test/miner_tests.cpp

```diff
@@ -508,13 +467,143 @@ BOOST_AUTO_TEST_CASE(CreateNewBlock_validity)
 
     BOOST_CHECK(pblocktemplate = AssemblerForTest(chainparams).CreateNewBlock(scriptPubKey));
     BOOST_CHECK_EQUAL(pblocktemplate->block.vtx.size(), 5U);
+}
+
+void MinerTestingSetup::TestPrioritisedMining(const CChainParams& chainparams, const CScript& scriptPubKey, const std::vector<CTransactionRef>& txFirst)
```

Comment:
Any reason you are passing chainparams? This isn't used and even it was, it wouldn't make sense to pass it, but use the chainparams of the existing chain from the TestingSetup context.